components/title/basic/src/index.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$c-title-basic: $c-white;
+$c-subtitle-basic: $c-white;
+
+$ff-roboto-sans-serif: 'Roboto', sans-serif; // Will be moved to nc-theme-basic
```

Comment:
We still need to clarify where the component vars should be really defined (here or themebasic). I any case I would name this var like  `$ff-title-basic: $ff-base` (actually $ff-base doesn't exist)